File: Python/1010 - Cálculo simples.py
produto_1 = input().split(' ')
c1 = int(produto_1[0])
n1 = int(produto_1[1])
v1 = float(produto_1[2])

produto_2 = input().split(' ')
c2 = int(produto_2[0])
n2 = int(produto_2[1])
v2 = float(produto_2[2])

total = n1 * v1 + n2 * v2

print('VALOR A PAGAR: R$ {:.2f}'.format(total))

File: Ruby/1010 - Cálculo simples.rb
p_1 = gets.split(' ')
n1 = p_1[1].to_i
v1 = p_1[2].to_f

p_2 = gets.split(' ')
n2 = p_2[1].to_i
v2 = p_2[2].to_f

total = n1 * v1 + n2 * v2

puts('VALOR A PAGAR: R$ %.2f' % total)